### ISSUE TITLE
users?

### ISSUE THREAD

**Issue by Bart274** [sbdomo/MyReadings#7]
what is the meaning of the users?
When I add a user and save my configuration, I get a message that my save succeeded.
But when I run my test I get an error stating that that user doesn't exist. When I open my myreadings-app I get the same error.

What are those users and how can we use them?

**Comment by sbdomo** [comment]
Now, for each user, you can save a bookmark of a book or indicate that the book has been read.
For that you must create a custom column in calibre for each user, the datatype must be integer.
For example, you create a custom column with label bart and name Bart274 and then in admin.php you add Bart274.
I recommend you to add, in the first one, a "guest" user who will be the account used by default.

**Comment by Bart274** [comment]
Now I don't have an error when I load myreadings, I used the wrong format for the columns :)

How can a user log in on myreadings?

**Comment by sbdomo** [comment]
It's not a log in. When you are in My readings (and you are connected if it's protected), you have to go in the configuration panel and you can choose the user.

**Comment by Bart274** [comment]
How do you choose a user?
I can't find that option anywhere...
the only thing I see is an option to change library if there are multiple libraries linked to my myreadings, and I see the profile I'm using at that moment.
![myreadings](https://cloud.githubusercontent.com/assets/1118630/2680515/8f718950-c183-11e3-8afd-caf603bd6eba.JPG)

**Comment by sbdomo** [comment]
Have you add and save the users with admin.php?
You shoud have something like that:
![config](https://cloud.githubusercontent.com/assets/2596665/2680753/1ddc298c-c186-11e3-8e1c-4412a0539bd2.jpg)

**Comment by Bart274** [comment]
Hmm...
When I try to save my configuration I just get a message 'error', with no other information.
But when I run the test on the second page, I see that all my users are OK. If I use a wrong user, I get an error for that user.
Any idea what might cause this?

**Comment by sbdomo** [comment]
The error message appear if the php server doesn't say that the file config.php has been saved. But I don't know why in your case.
If you reload admin.php, did you see the correct config in the page ?

**Comment by Bart274** [comment]
Yeah, everything is saved, even if I view that file from another computer on another network... :s 
You could check out my myreadings server at: http://bart274.mooo.com:8888/myreadings/#

**Comment by sbdomo** [comment]
I can't try it now. I'll do some tests with your url later...

**Comment by Bart274** [comment]
ok, thanks in advance :)

**Comment by sbdomo** [comment]
Hello,
I have tested your site but I didn't found problem. I can see the users as in that image:
![image2](https://cloud.githubusercontent.com/assets/2596665/2686472/affeaaea-c205-11e3-8762-583e4c5a5cad.jpg)

But, it seems the admin.php have a problem of cache (an old config.php seems sometimes used and I think it's because of cache). I look for a solution. It's perhaps an explanation: it's now the good version of your config.php...

There is a new commit with some bugfix and new features...

**Comment by Bart274** [comment]
Hmm, it was a caching problem! On other pc's I see the new configuration! :)
